app/templates/briefs/view_response_result.html

```diff
@@ -148,8 +148,8 @@ <h2 class="summary-item-heading">What happens next</h2>
 
   {%
     with
-      url = url_for(".dashboard"),
-      text = "Your account"
+      url = url_for(".opportunities_overview", framework_slug=brief.frameworkSlug),
```

Comment:
And then in the code we could call it `opportunities_dashboard`, to mirror the `dashboard` and `framework_dashboard` that we already have.